Main/gyro.ino

```diff
@@ -164,15 +164,14 @@ void detectThrow() {
 //if statement, if the total value is greater than 8000, we assume that the device is moving//
            if(totalValue >8000){
             //if device is moving, the lights turn on//
-            digitalWrite(lightZapper, HIGH);
-            Serial.print("HIGH");
-            Serial.print("\n");
+            return true;
             
            //if device is not moving, the lights turn off//
            }else{
-            digitalWrite(lightZapper, LOW);
-            Serial.print("LOW");
-            Serial.print("\n");
+            //digitalWrite(lightZapper, LOW);
+            //Serial.print("LOW");
+            //Serial.print("\n");
+            return false;
```

Comment:
Great work here guys, just need to watch the indentation of your code because there's quite a large gap in lines 141 - 174